packages/statemanager/tests/stateManager.spec.ts

```diff
@@ -24,7 +23,6 @@ tape('StateManager', (t) => {
     const stateManager = new DefaultStateManager()
 
     st.deepEqual(stateManager._trie.root, KECCAK256_RLP, 'it has default root')
-    st.equal(stateManager._common.hardfork(), 'petersburg', 'it has default hardfork')
```

Comment:
How come this defaulted to `petersburg` and this test passed!?

Reply:
:thinking: No idea but it's gone now :-)